samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/FlipViewSamplePage.xaml.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.Toolkit.Samples.Entities;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Uno.Toolkit.Samples.Content.NestedSamples;
+using Uno.Toolkit.UI;
+using Uno.Toolkit.Samples.Helpers;
+
+#if __IOS__
+using UIKit;
+#endif
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Navigation;
+using XamlWindow = Microsoft.UI.Xaml.Window;
+
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using XamlWindow = Windows.UI.Xaml.Window;
+using System.Collections.ObjectModel;
+
+
+#endif
+
+namespace Uno.Toolkit.Samples.Content.Controls
+{
+	[SamplePage(SampleCategory.Behaviors, nameof(FlipViewExtensions))]
+	public sealed partial class FlipViewSamplePage : Page
+	{
+		public FlipViewSamplePage()
+		{
+			this.InitializeComponent();
+
+
+			var btn = (Button)FindName("btn");
```

Comment:
there is no guarantee that the data-template is materialized yet
wrap the whole thing in
```cs
this.Loaded += (s, e) =>
{
	// mark the <SamplePageLayout /> in the xaml with x:Name="SamplePageLayout"
	var button = SamplePageLayout.GetSampleChild<Button>(Design.Asgnostic, "btn");
	...
};
```